tensorflow/python/keras/layers/preprocessing/image_preprocessing.py

```diff
@@ -1323,14 +1323,17 @@ def _create_variable(self, *args, **kwargs):
     # that it skips the distribution-strategy check. When we are inside a
     # distribution-strategy scope, variables.Variable will pick a proper
     # variable class (e.g. MirroredVariable).
-    return variables.Variable(*args, **kwargs)
+    v = variables.Variable(*args, **kwargs)
+    K.track_variable(v)
+    return v
```

Comment:
Do you still need this?

Reply:
Is this comment addressed?

Reply:
I think this is needed, as the method override `_create_variable` in stateful_random_ops.Generator